testmodule/fileparser.py

```diff
@@ -10,12 +11,12 @@
 class Rule:
     """It defines the class Rule."""
 
-    def __init__(self, group, number, logic):
+    def __init__(self, group: str, number: int, logic: str):
```

Comment:
`logic` is a function, this is annotated using `Callable`

https://docs.python.org/3/library/typing.html#annotating-callable-objects